### PR TITLE
AC-682 added a calendar Icon

### PR DESCRIPTION
**Pull request by LuGO0**
## Description of what I changed
added a calendar icon instead of a simple green button for the AddEditPatientFragment

## Issue I worked on
JIRA Issue: https://issues.openmrs.org/projects/AC/issues/AC-682

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by LuGO0** [comment]
attached screenShots for reference :)

**Comment by LuGO0** [comment]
@f4ww4z please review this Sir!!

File: openmrs-client/src/main/res/layout/fragment_patient_info.xml [hunk 1]
```diff
@@ -327,7 +327,7 @@
                             android:layout_marginTop="8dp"
                             android:layout_marginEnd="24dp"
                             android:layout_marginRight="24dp"
-                            android:background="@drawable/ic_date_range_24dp" />
+                            android:background="@drawable/ic_calendar" />
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/layout/fragment_patient_info.xml]: @LuGO0 There's no need to change this, it is enough to just change `Button` to `ImageButton` as per line 320. See #697

**Comment by LuGO0** [comment]
@f4ww4z Sir I have made the requested changes.

**Pull request merged**